### PR TITLE
Feat: Create README for identity service

### PR DESCRIPTION
**Pull request by ZixMai**
1. Created README for identity service

### CODE REVIEW

**Review by MaxPlays35 — APPROVED** [review]

File: identity_service/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+___
+# Identity service
+[![Build identity service](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_identity_service.yaml/badge.svg)](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_identity_service.yaml)
+___
+**Установка и описание**
+___
+1. **Образец .env файла**
+	```
+	DATABASE_URL="DATABASE_URL"
```
Comment (Nhsdkk) [inline identity_service/README.md:14]: Wrong env format. Should be 
```
ACCESS_SECRET = "***"
REFRESH_SECRET = "***"
DATABASE_URL = "***"
ENVIRONMENT = "***"
ACCESS_TOKEN_EXPIRATION = 15
REFRESH_TOKEN_EXPIRATION = 30
```

File: identity_service/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+		```
+    	docker pull ghcr.io/unspokenteam/identity_service:latest
+ 		```
+       - Создайте .env файл, укажите его путь в команде и выполните её:
+		```
+		docker run -d -p 8080=8080 —env-file $PATH_TO_ENV ghcr.io/unspokenteam/identity_service:latest
+ 		```
+
+	2. **Через poetry**
+        - Также возможен локальный запуск при помощи утилиты poetry **без использования docker'a**. Для этого введите и выполните команду:
```
Comment (Nhsdkk) [inline identity_service/README.md]: Add generate proto and prisma steps

File: identity_service/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        poetry run python main.py
+        ```
+---
+3. **О микросервисе**:
+	- Сервис полностью осуществляет взаимодействие с аккаунтами пользователей в календаре.
+	- Также под контролем сервиса находится взаимодействие со всеми данными аккаунта пользователя и токенами.
+	- Функционал:
+        - **login** - **Получение токенов при успешной авторизации**
+
+        - **register** - **Создание токенов при регистрации пользователя**
```
Comment (Nhsdkk) [inline identity_service/README.md]: Just user registration

File: identity_service/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+        - **auth** - **Получение ID аунтефицированного пользователя**
```
Comment (Nhsdkk) [inline identity_service/README.md]: Just retrieval of user data

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Make changes

File: identity_service/README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+___
+# Identity service
+[![Build identity service](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_identity_service.yaml/badge.svg)](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_identity_service.yaml)
+___
+**Установка и описание**
+___
+1. **Образец .env файла**
+	```
+	DATABASE_URL="DATABASE_URL"
+	ENVIRONMENT="ENVIRONMENT"
```
Comment (Nhsdkk) [inline identity_service/README.md:16]: Add comments about variables

File: identity_service/README.md [hunk 6]
```diff
@@ -6,9 +6,20 @@ ___
...
 1. **Образец .env файла**
 	```
+    ACCESS_SECRET="ACCESS_SECRET"
+    REFRESH_SECRET="REFRESH_SECRET"
 	DATABASE_URL="DATABASE_URL"
 	ENVIRONMENT="ENVIRONMENT"
+    REFRESH_TOKEN_EXPIRATION=30
+    ACCESS_TOKEN_EXPIRATION=15
 	```
+    - где:
```
Comment (Nhsdkk) [inline identity_service/README.md]: Its better to use comments inside env example

File: identity_service/README.md [hunk 7]
```diff
@@ -6,9 +6,20 @@ ___
...
+        - ACCESS_SECRET - Ключ для генерации Access token
+        - REFRESH_SECRET - Ключ для генерации Refresh token
+        - DATABASE_URL - адрес базы данных
+        - ENVIRONMENT - вид окружения
+        - REFRESH_TOKEN_EXPIRATION - Время истечения срока валидности Refresh token в днях
+        - ACCESS_TOKEN_EXPIRATION - Время истечения срока валидности Access token в минутах
 ___
 2. **Установка**:
```
Comment (Nhsdkk) [inline identity_service/README.md:23]: Add setup database step

File: identity_service/README.md [hunk 8]
```diff
@@ -39,9 +58,9 @@ ___
 	- Функционал:
         - **login** - **Получение токенов при успешной авторизации**
 
-        - **register** - **Создание токенов при регистрации пользователя**
-
-        - **auth** - **Получение ID аунтефицированного пользователя**
+        - **register** - **Регистрация пользователя**
+poetry run python -m grpc_tools.protoc -I ../shared/proto --python_out=generated --grpc_python_out=generated --pyi_out=generated ../shared/proto/user/*.proto ../shared/proto/notification_service/*.proto && poetry run protol --create-package --in-place --python-out generated protoc --experimental_allow_proto3_optional --proto-path=../shared/proto ../shared/proto/notification_service/*.proto ../shared/proto/user/*.proto
```
Comment (Nhsdkk) [inline identity_service/README.md]: wtf is this

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Make changes

**Review by Nhsdkk — APPROVED** [review]
LGTM

**Pull request merged**